proto/internalapi/sensor/collector.proto

```diff
@@ -24,5 +24,11 @@ message NetworkConnectionConfig {
 // At this time it only controls if external IPs are aggregated at the
 // cluster level.
 message CollectorConfig {
+  message ExternalIPs {
+    bool enable = 1;
+  }
+  message Networking {
```

Comment:
I think `NetworkingConnection` might be better so that it is separate from `NetworkingEndpoint`.